source/nodejs/package.json

```diff
@@ -5,22 +5,34 @@
 	"scripts": {
 		"start:designer": "lerna run start --scope=adaptivecards-designer --stream",
 		"start:designer-app": "lerna run start --scope=adaptivecards-designer-app --stream",
-		"start:all": "lerna run --parallel watch && lerna run --parallel start",
-		"audit-all": "lerna-audit",
-		"format": "clang-format-launcher --verbose",
-		"verify": "clang-format-launcher -verify --verbose",
-		"prepare": "cd ../.. && husky install source/nodejs/.husky"
+		"start:all": "lerna run --parallel watch && lerna run --parallel start"
```

Comment:
will doing auditing be same?

Reply:
auditing should be simpler now -- just `npm audit`